scripts/utils.ts

```diff
@@ -27,7 +27,7 @@ export function execAsync(
       const childProcess: ChildProcess = exec(command, { env });
 
       childProcess.on('exit', (code: number) => {
-        if (code !== 0 || stderrStream.toString().length) {
```

Comment:
Terraform [outputs debug logs to stderr](https://developer.hashicorp.com/terraform/internals/debugging). We can target that specific case if there's a specific reason why this was present.